client/signup/steps/add-ons/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { MinimalRequestCartProduct } from '@automattic/shopping-cart';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import AddOnsGrid from 'calypso/my-sites/add-ons/components/add-ons-grid';
+import useAddOns, { AddOnMeta } from 'calypso/my-sites/add-ons/hooks/use-add-ons';
+import NavigationLink from 'calypso/signup/navigation-link';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+import './styles.scss';
+
+interface Props {
+	stepSectionName: string | null;
+	stepName: string;
+	goToStep: () => void;
+	goToNextStep: () => void;
+	flowName: string;
+	positionInFlow: number;
+	defaultDependencies: object;
+	forwardUrl: string;
+}
+interface AddOnsProps {
+	selectedAddOns: string[];
+	addOns: ( AddOnMeta | null )[];
+	onToggleAllAddOns: () => void;
+	onAddAddon: ( addonSlug: string ) => void;
```

Comment:
```suggestion
	onAddAddOn: ( addOnSlug: string ) => void;
```

Nitpick - no need to update. It happens in a few places, but mainly curious - should we camelCase `addon` -> `addOn`? Not sure if I'm getting it wrong instead! 😬